bears/general/LineCountBear.py

```diff
@@ -10,12 +10,13 @@ class LineCountBear(LocalBear):
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Formatting'}
 
-    def run(self, filename, file, max_lines_per_file: int):
+    def run(self, filename, file, max_lines_per_file: int,
+            min_lines_per_file: int):
```

Comment:
Add a default `min_lines_per_file: int=0` , then you dont need to modify `.coafile`